.github/workflows/label-actions.yml

```
@@ -4,6 +4,9 @@ on:
   issues:
     types: [labeled]
 
+permissions:
+  issues: write
```

```suggestion
  contents: read
  issues: write
```

Ok, adding `contents: read`